xgettext.js

```diff
@@ -164,7 +164,9 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 
 	traverse( parsedInput.ast, {
 		CallExpression: function( node ) {
-			var functionName = ( node.callee.property ) ? node.callee.property.name : node.callee.name;
+			// Pull the resultingFunction out of (0, resultingFunction)()
+			var callee = ( node.callee.type === 'SequenceExpression' ) ? _.last( node.callee.expressions ) : node.callee;
```

Comment:
Can you explain what led you to the `SequenceExpression` type? From the [parser spec](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Parser_API#Expressions), this seems like a reasonable choice for supporting fat arrow functions, but I fail to see why it fixes the particular syntax we're accounting for here.